graphdatascience/graph_data_science.py

```diff
@@ -32,7 +32,31 @@ def __init__(
         auth: Optional[Tuple[str, str]] = None,
         aura_ds: bool = False,
         arrow: bool = True,
+        arrow_disable_server_verification: bool = True,
```

Comment:
This constructor is starting to get out of hand. The idea of using the same constructor for arrow was to hide it from the user, but that's no longer happening anyway. I could imagine having something like a static `GraphDataScience.with_tls` constructor. But I don't know

Reply:
I agree. I mean .. the `disable_server_verification` doesn't necessarily be exposed, but some builder or factory methods would be nice. We need to make some more changes wrt Arrow config, I can try to sketch something on that PR.